catalog/imaging/imaging-actionprovider-chip/src/main/java/org/codice/alliance/imaging/chip/actionprovider/ImagingChipActionProvider.java

```diff
@@ -78,12 +79,19 @@ public String getId() {
     public <T> boolean canHandle(T subject) {
         if (subject instanceof Metacard) {
             Metacard metacard = (Metacard) subject;
-            String contentTypeName = metacard.getContentTypeName();
-            if (NITF_CONTENT_TYPE.equals(contentTypeName) && metacard.getLocation() != null) {
-                return true;
+
+            Attribute attribute = metacard.getAttribute(Media.TYPE);
```

Comment:
It seems to me that the canHandle() method should be checking if the metacard is an image NITF, has a location, and has the expected derived images.